initfiles/sbin/add_conf_settings.sh.in

```diff
@@ -33,16 +33,63 @@ ${USER_NAME} ALL = NOPASSWD: HPCC_DAFILESRV, HPCC_HPCCINIT
 Defaults:${USER_NAME} !requiretty
 %EOF
 
+limits=(
+  MIN_SOFT_nofile="8192"
```

Comment:
Why make the soft limit only 8k ?  The scripts used to bump this to the hard limit always, right ?